src/App.js

```diff
@@ -1,5 +1,90 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+
+  getRandomNumber(){
+    const computerNumber = [];
+    while (computerNumber.length < 3){
+      const number = MissionUtils.Random.pickNumberInRange(1, 9);
+      if(!computerNumber.includes(number))
+        computerNumber.push(number);
+    }
+    return computerNumber;
+  }
+
+  async getUserInput(){
+    const userInput = await MissionUtils.Console.readLineAsync("숫자를 입력해주세요 : ");
+    const validatedUserInput = this.checkInputValidation(userInput);
+
+    return validatedUserInput;
+  }
+
+  checkInputValidation(userInputNumber){
+    const userInputNumberArr = userInputNumber.split('').map(Number);
+
+    if(userInputNumber.length !== 3) throw new Error("[ERROR] 3자리 숫자가 아닙니다.");
+    if(new Set(userInputNumberArr).size !== 3) throw new Error("[ERROR] 중복된 숫자가 있습니다.");
+    if(userInputNumber.includes(0)) throw new Error("[ERROR] 1~9 사이의 숫자가 아닙니다.");
+    if(isNaN(userInputNumber)) throw new Error("[ERROR] 숫자가 아닙니다.");
+    
+    return userInputNumberArr;
+  }
+
+  compareNumber(computerNumber, userInput){
+    let strikeCount = 0;
+    let ballCount = 0;
+
+    for(let i = 0; i < 3; i++){
+      if(computerNumber[i] === userInput[i]) strikeCount++;
+      else if(computerNumber.includes(userInput[i])) ballCount++;
+    }
+    return {strikeCount, ballCount};
+  }
+
+  async answerResult(computerNumber, userInput){
+    if(computerNumber.join('') === userInput.join('')){
+      MissionUtils.Console.print("3스트라이크");
+      return true;
+    } else {
+      const STRIKE_COUNT = this.compareNumber(computerNumber,userInput).strikeCount;
+      const BALL_COUNT = this.compareNumber(computerNumber,userInput).ballCount;
+
+      if(BALL_COUNT > 0 && STRIKE_COUNT > 0) MissionUtils.Console.print(`${BALL_COUNT}볼 ${STRIKE_COUNT}스트라이크`);
+      else if(BALL_COUNT > 0 && STRIKE_COUNT === 0) MissionUtils.Console.print(`${BALL_COUNT}볼`);
+      else if(BALL_COUNT === 0 && STRIKE_COUNT > 0) MissionUtils.Console.print(`${STRIKE_COUNT}스트라이크`);
+      else if(BALL_COUNT === 0 && STRIKE_COUNT === 0) MissionUtils.Console.print(`낫싱`);
+      
+      return false;
+    }
+  }
+
+  async startGame(){
+    const computerAnswer = this.getRandomNumber();
+    let gameResult = false;
+
+    while(!gameResult){
```

Comment:
이곳도 변수명만 보고 역할, 타입을 유추하기 쉽게 작성하면 좋을 것 같아요
`boolean` 타입에는 `is변수명`, `has변수명` 처럼 작성하는 것을 추천합니다!

https://github.com/airbnb/javascript#accessors--boolean-prefix

```suggestion
// 변수명은 참고만 해주세요!
    let isReplay = true;

    while(isReplay){
```